backend/src/main/java/de/echochamber/backend/controller/PollController.java

```diff
@@ -32,12 +30,14 @@
     private final PollService pollsService;
     private final UserService userService;
     private final AnswerService answerService;
+    private final Mapper mapper;
 
     @Autowired
-    public PollController(PollService pollsService, UserService userService, AnswerService answerService) {
+    public PollController(PollService pollsService, UserService userService, AnswerService answerService, Mapper mapper) {
         this.pollsService = pollsService;
         this.userService = userService;
         this.answerService = answerService;
+        this.mapper = mapper;
     }
 
     @PostMapping(value = "/create", produces = APPLICATION_JSON_VALUE, consumes = APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
    @PostMapping(value = "/", produces = APPLICATION_JSON_VALUE, consumes = APPLICATION_JSON_VALUE)
```
Create a new entity by posting to the collection